La-Z-Boy/__main__.py

```diff
@@ -45,6 +77,11 @@ def getBSoup(url):
     soup = BeautifulSoup(req.read(), "lxml")
     return soup
 
+def getBSoup2(url):
+    req = urllib2.urlopen(url)
+    soup = BeautifulSoup(req.read(), "lxml")
+    return soup
+
```

Comment:
The thing is , getBSoup() also prints text '     Processing ..... ' that I did not want as I am calling the function multiple times.   
So I just created a copy as getBSoup2() which just does not print this text